apps/src/hamburger/hamburger.js

```diff
@@ -68,5 +71,16 @@ export const initHamburger = function() {
         trackEvent('help_ui', 'support', 'studio_footer');
       });
     });
+
+    let channelId = getChannelIdFromUrl(location.href);
+    const userAlreadyReportedAbuse =
```

Comment:
looks like we're currently using this logic in three (?) different places. Can it be pulled out into some reusable code, and/or removed from some of those places?

Reply:
I created reportAbuse.js and moved the shared logic to there.